app/src/main/java/org/mozilla/fenix/addons/AddonDetailsFragment.kt

```diff
@@ -107,7 +107,7 @@ class AddonDetailsFragment : Fragment(R.layout.fragment_add_on_details) {
     private fun bindAuthors(addon: Addon, view: View) {
         view.author_text.text = addon.authors.joinToString { author ->
             author.name + " \n"
```

Comment:
I think it will be better to remove `" \n"` that we are using as a separator, as [joinToString](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/join-to-string.html) has a `separator: CharSequence = ", "` default, it is OK the `trim` as  this is formation comes from an external source.